apps/site/src/app/(home)/sections/Countdown/Countdown.module.scss

```diff
@@ -0,0 +1,127 @@
+.paraboloid {
+	position: relative;
+	width: 100%;
+}
+
+.countdownWrapper {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	position: relative;
+}
+
+.descriptiveText {
+	top: 45px;
+	left: 20px;
+	position: absolute;
+	min-width: 150px;
+	transform: translateX(-50%);
+	text-align: center;
+}
+
+.countdownMaterial {
+	// margin-top: 10%;
+	position: absolute;
+	top: 0px;
+	left: 0px;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+
+@media only screen and (max-width: 400px) {
+	.clockPos {
+		margin-top: 45%;
+	}
+	.preCountdown {
+		margin-top: 50%;
+	}
+	.endText {
+		margin-top: 50%;
+	}
+}
+
+@media only screen and (min-width: 401px) and (max-width: 600px) {
+	.clockPos {
+		margin-top: 30%;
+	}
+	.preCountdown {
+		margin-top: 40%;
+	}
+	.endText {
+		margin-top: 40%;
+	}
+}
+
+@media only screen and (min-width: 601px) and (max-width: 1000px) {
+	.clockPos {
+		margin-top: 20%;
+	}
+	.preCountdown {
+		margin-top: 20%;
+	}
+	.endText {
+		margin-top: 20%;
+	}
+}
+
+@media only screen and (min-width: 1001px) {
+	.preCountdown {
+		margin-top: 15%;
+	}
+	.endText {
+		margin-top: 10%;
+	}
+}
+
+.clockPos {
+	width: 100%;
+	display: flex;
+	justify-content: center;
+}
+
+.boat {
+	position: absolute;
+	transform: translate(-50%, 40%);
+}
+
+.outerCircle {
+	background-color: #ffffff;
+	// border-color: #1a1840;
+	border: 5px solid #1a1840;
```

Comment:
Chore (repeating): The color here can be replaced with `theme.$lighter-black` from the theme scss file